CredentialProvider/CredentialWelcomeViewController.swift

```diff
@@ -5,22 +5,73 @@
 import UIKit
 
 class CredentialWelcomeViewController: UIViewController {
-
-    @IBOutlet weak var taglineLabel: UILabel! {
-        didSet {
-            taglineLabel.text = .LoginsWelcomeViewTitle
+    
+    lazy private var logoImageView: UIImageView = {
+        let logoImage = UIImageView(image: UIImage(named: "logo-glyph"))
+        return logoImage
+    }()
+    
+    lazy private var taglineLabel: UILabel = {
+        let label = UILabel()
+        label.text = .LoginsWelcomeViewTitle
+        label.font = UIFont.systemFont(ofSize: 15)
+        return label
+    }()
+    
+    lazy private var activityIndicator: UIActivityIndicatorView = {
+        let loadingIndicator = UIActivityIndicatorView()
+        if #available(iOSApplicationExtension 13.0, *) {
+            loadingIndicator.style = .large
         }
+        return loadingIndicator
+    }()
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
     }
     
-    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        view.backgroundColor = UIColor(named: "launchScreenBackgroundColor")
+        addSubviews()
+        addViewConstraints()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
         activityIndicator.startAnimating()
     }
     
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
         activityIndicator.stopAnimating()
     }
+    
+    func addSubviews() {
+        view.addSubview(logoImageView)
+        view.addSubview(taglineLabel)
+        view.addSubview(activityIndicator)
+    }
+    
+    func addViewConstraints() {
+        view.addSubview(logoImageView)
```

Comment:
Looks like `logoImageView` is added twice.

Reply:
It was a mistake, I will remove the duplicate. Thank you